src/api/cpp/cvc5_kind.h

```diff
@@ -1287,6 +1287,23 @@ enum Kind : int32_t
    *   - Solver::mkOp(Kind, const std::vector<uint32_t>&) const
    */
   BITVECTOR_NEG,
+  /**
+   * Unsigned addition overflow detection.
+   *
+   * - Arity: ``2``
+   *
+   *   - ``1..2:`` Terms of bit-vector Sort (sorts must match)
+   *
+   * - Create Term of this Kind with:
+   *
+   *   - Solver::mkTerm(Kind, const std::vector<Term>&) const
+   *   - Solver::mkTerm(const Op&, const std::vector<Term>&) const
+   *
+   * - Create Op of this kind with:
+   *
+   *   - Solver::mkOp(Kind, const std::vector<uint32_t>&) const
```

Comment:
We can't create an Op with this kind, do we?

Reply:
We can create Ops of any kind.